provider/docker.go

```diff
@@ -270,7 +270,12 @@ func (provider *Docker) getIPAddress(container dockertypes.ContainerJSON) string
 			}
 		}
 	}
-	for _, network := range container.NetworkSettings.Networks {
+	for networkName, network := range container.NetworkSettings.Networks {
```

Comment:
Well, `host` is a reserved named for network (to *map* the networkMode host) but well…
The other thing is on versions, `networkMode` is guaranteed to be there since… well a while…, but `NetworkSettings.Networks` has been updated quite a while.

Anyway that's just a nit, backward compatibility should make it that it works ok like that for more than a while :angel: